cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -558,3 +558,59 @@ def test_vm_notes_ui(appliance, provider):
     # to check it against what's displayed in the UI.
     assert view.basic_information.read()['Notes'] != '', ("VM notes field is empty in CFME or"
         "VM does not have any notes")
+
+
+@pytest.mark.meta(automates=[1644770])
+@pytest.mark.parametrize('create_vm', ['small_template'], indirect=True)
+def test_delete_vm_archive(appliance, provider, create_vm):
+    """
+    Bugzilla:
+        1644770
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/2h
+        testtype: functional
+        testSteps:
+            1.Add VMware provider to CFME
+            2.Create a new Vm
+            3.Remove it from vmware using delete operation and check status of that vm in CFME -
+            should be archived
+        expectedResults:
+            1.VMware provider added successfully.
+            2.New VM visible in CFME
+            3.VM is Archived in CFME
+    """
+    create_vm.mgmt.delete()
+    create_vm.wait_for_vm_state_change(from_any_provider=True, desired_state='archived')
```

Comment:
Can we wrap this in a try/except to call `pytest.fail` on the TimedOutError exception please?